### ISSUE TITLE
Outgoing multisigned transactions show as incoming

### ISSUE THREAD

**Issue by nemster** [NemProject/NEMAndroidApp#294]
I have one of the cosigner of a multisigned account of my android wallet.

Outgoing transactions from the multisigned account are green and have a + sign as it they were incoming.

**Comment by nemster** [comment]
Android 7.0 on Huawei P9 Lite

NEM Wallet 1.0.72_mainnet(111)

I have a 2 of 3 multisig account; one of the tree cosigners is on my android wallet.
I create an outgoing transaction on another device, I confirm the transaction from the android NEM wallet.
The transaction appears in the wallet history with a + sign even if it was an outgoing transaction.

**Comment by EvertonMelo** [comment]
@nemster 

Great! 

let's go with for little steps ok? 

I you have 2 of 3 multsig account with your NemAndroid Wallet ok? 

> I create an outgoing transaction on another device.

( example other NemadroidWallet ou NanoWallet with Desktop, explait more for ti please. )

>I confirm the transaction from the android NEM wallet.
( you NamAdroid Wallet or other too multsig? ) 

I began to suspect that there is incompatibility between some versions of the multsig.

**Comment by nemster** [comment]
Yes, first cosigner was NanoWallet; I sent the transaction from it.

I open NemAndroid Wallet, select the cosigner account, and it tells me there's a transaction waiting for my signature.

I sign the transaction and it appears with the + sign

**Comment by EvertonMelo** [comment]
@nemster 

I need the data from Nanowallet: version, Built; 

I see that there is an incompatibility between some transactions between:
nanowallet, Nemandroid, NemIOSapp. 

could you do a test using only Nanowallet? 
that is, one installation creates the transaction other than the other.

@saulgray @gimer 

Can you confirm that this is the case?

**Comment by nemster** [comment]
Initiator was NanoWallet 1.4.13

**Comment by EvertonMelo** [comment]
@nemster 

the NanoWallet 1.4.13 not support Multsign:

# **Changelog NanoWallet** 1.4.13:

    Allow sub-namespaces and mosaic names to start with a number
    Fix secondary accounts: In occasional cases, secondary accounts private keys can start with '00' but BIP32 was omitting it, giving a 62 characters private key. Those secondary accounts will fail to send anything out even if password is valid (but recoverable).
    Update languages

but the 
# **NanoWallet 2.2.0 Changelog**

    _Create and verify signed messages modules_
    Account transactions explorer
    Fix timestamp too far in the future
    Updated voting backend
    Minor fixes and improvements

I need update for the version 2.2.0 for the support the Feature.
